packages/ui/src/CodeEditor.module.scss

```diff
@@ -0,0 +1,17 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.container {
```

Comment:
@utkukaratas could you get @pawelklasa to sign off on the colors and fonts?

Reply:
pawel gives the green. resolving.